### PR TITLE
Ptmap.2.0.1

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/ptmap/ptmap.2.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  ["obuild" "build" "lib-ptmap"]
+]
+install: [
+  ["obuild" "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "ptmap"]
+]
+depends: [
+  "ocamlfind"
```
Comment (dsheets) [inline packages/ptmap/ptmap.2.0.1/opam]: This can probably be marked `{build}`? This should not block merging.

**Comment by UnixJunkie** [comment]
@dsheets  honestly, this package only needs ocamlfind to uninstall.
Does this really qualifies as {build} ?

**Comment by dsheets** [comment]
`{build}` is weaker than nothing so right now you are claiming that you require `ocamlfind` at run-time as well.

**Comment by UnixJunkie** [comment]
ok, done.

**Pull request merged**